backend/scripts/db.py

```diff
@@ -88,6 +76,23 @@ def AddNewItem(Data):
             .format(f"LMR-{(id-1):04}" ,id))
     commit()
 
+def GetItemsWithIssues():
+    IDs = []
+    execute("select id from items where price = 999;")
+    data = cur.fetchall()
+    for item in data:
+        IDs.append(item[0])
+    execute("select  id from items where cnt < items.min_cnt;")
+    data = cur.fetchall()
+    for item in data:
+        IDs.append(item[0])
+    return IDs
+
+def UpdateItem(ItemData: Interfaces.Item):
+    execute("update items set (name, cnt, price, manufacturer, color, min_cnt, details, size, bon_name) = "+
+            "('{}',{},{},'{}','{}',{},'{}','{}','{}') where id = {}"
```

Comment:
use f"{variable}, {variable}" for clarification. (More easily to understand that way)